doc/source/conf.py

```diff
@@ -424,6 +424,29 @@ def add_custom_assets(
         app.add_css_file("css/ray-libraries.css")
 
 
+def set_examples_redirects(
+    app: sphinx.application.Sphinx,
+    pagename: str,
+    templatename: str,
+    context: Dict[str, Any],
+    doctree: nodes.Node,
+):
+    """Insert JS which redirects users to the example gallery for certain pages."""
+    tags = {
+        "ray-core/examples/overview": "",
```

Comment:
I think we need a Ray Core tag to make this work

Reply:
@angelinalg What should the tags be for each of these sections?

Reply:
I thought there was talk about a tag for each library. Something like `core`, `data`, `serve`, `train`, `rllib`, `cluster`?

Reply:
Nobody went through the examples in the example gallery systematically to assign these tags. I started putting them on examples I touched (and was sure about) for other PRs a while back, anticipating that we'd want to do this at some point.

Reply:
I think there are already tags that correspond to the different libraries. Can we just use those? Eventually, we may need a second set: one for the library and one for the workload.

Reply:
Fixed.